### PR TITLE
Adding a New Split Activation Test with VRS SDK

### PR DESCRIPTION
**Pull request by vareti**

### CODE REVIEW

File: api/NuageAPI_test.go [hunk 1]
```diff
@@ -959,3 +959,91 @@ func TestVMPowerOff(t *testing.T) {
...
 	vrsConnection.Disconnect()
 }
+
+//TestSplitActivation tests the split activation mode using SDK
+func TestSplitActivation(t *testing.T) {
+	vrsConnection, err1 := NewUnixSocketConnection(UnixSocketFile)
+	if err1 != nil {
+		t.Fatal("Unable to connect to the VRS")
+	}
+	enterpriseFetchingInfo := &bambou.FetchingInfo{Filter: "name == \"" + Enterprise + "\""}
```
Comment (rparulek) [inline api/NuageAPI_test.go]: We have a util package that does fetching enterprises task for you; I would use that. Similar for domains and subnets.

Comment (vareti) [reply api/NuageAPI_test.go]: made changes addressing your comments

**Comment by vishpat** [comment]
@vareti Changes look great, can you please rebase and create 1 commit. I will get the changes merged in.

**Comment by vareti** [comment]
@vishpat rebased the repo and created only one commit

**Pull request merged**